### PR TITLE
my domain

### PR DESCRIPTION
**Pull request by svg-rs**
my domain

### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://github.com/svg-rs/svg.github.io

### CODE REVIEW

**Review by svg-rs — COMMENTED** [review]
ok

File: domains/svg.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "svg's website",
+    "repo": "https://github.com/svg-rs/svg.github.io",
+    "owner": {
+      "username": "svg-rs",
+      "email": "kimmkayy8@proton.me"
+    },
+    "record": {
+      "CNAME": "svg.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/svg.json:9]: Please change the record value to `svg-rs.github.io`.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/svg.json (hunk 1 above)
Comment (svg-rs) [reply domains/svg.json:9]: ok

Comment (svg-rs) [reply domains/svg.json]: i fixed it
can you merge noww

**Review by wdhdev — APPROVED** [review]
I cannot merge PR's as I'm only a helper.

**Review by TheBotlyNoob — APPROVED** [review]
:+1:

**Pull request merged**